lib/gman_client.rb

```diff
@@ -8,4 +8,14 @@ def self.drivers
     respond = request.api.v1.drivers.get
     respond.map(&:to_h)
   end
+  def self.driver_commission_histories
+    request = Blanket.wrap("http://localhost:3000", :extension => :json)
```

Comment:
Ruby Gems start out with modules. So something like:
```Ruby
Module GmanClient
  Class Retrieve
    def initialize(host = nil)
      fail ArgumentError.new('host is required') if host.nil?

      @host = host
    end

     def drivers
        response = request.api.v1.drivers.get
        response.map(&:to_h)
     end

  # other API methods

  def request
    Blanket.wrap(@host, :extension => :json)
  end
end
```
might work out. I do like the ideas of initializers testing to make sure the environments exist. Im thinking we can also override the url/secret/application id for custom calls or per application.